### PR TITLE
Fix test_cacl_application for latest change in caclmgrd

### PR DESCRIPTION
**Pull request by ZhaohuiS**
### Description of PR

Summary:
Fixes # (issue)
According PR https://github.com/sonic-net/sonic-host-services/pull/139, the expected iptables rules are changed.

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [ ] 202405

#### What is the motivation for this PR?
RP test will fail after https://github.com/sonic-net/sonic-host-services/pull/139, update the case accordingly.
#### How did you do it?
add expected iptable rules

#### How did you verify/test it?
run cacl/test_cacl_application.py

### CODE REVIEW

**Review by maipbui — APPROVED** [review]

File: .azure-pipelines/pr_test_scripts.yaml [hunk 1]
```diff
@@ -19,7 +19,6 @@ t0:
   - bgp/test_bgp_speaker.py
   - bgp/test_bgp_update_timer.py
   - bgp/test_bgpmon.py
-  - cacl/test_cacl_application.py
```
Comment (wangxin) [inline .azure-pipelines/pr_test_scripts.yaml:22]: With the skip rule in conditional mark, probably we can keep this line here to include in in PR testing. Then after the issue is fixed and resolved, then this script will be automatically covered by PR testing.

Comment (ZhaohuiS) [reply .azure-pipelines/pr_test_scripts.yaml:22]: @wangxin In my second commit, I  added the skip condition and kept this line, but PR test still kept failing with this case, that's why I removed this line, I will add this line back when image change is included.

**Review by wangxin — APPROVED** [review]

**Comment by yutongzhang-microsoft** [comment]
@ZhaohuiS Why you remove the script `cacl/test_cacl_application.py` from PR test?

**Comment by weiguo-nvidia** [comment]
Hi @ZhaohuiS 

May I ask is there any plan to cherry-pick to 202405 branch?

**Comment by ZhaohuiS** [comment]
> Hi @ZhaohuiS
> 
> May I ask is there any plan to cherry-pick to 202405 branch?

@weiguo-nvidia  It has been in 202405 already. https://github.com/sonic-net/sonic-mgmt/pull/13900

**Pull request merged**